app/src/main/res/values/strings.xml

```diff
@@ -103,7 +103,7 @@
     <string name="content_descriptor_edit_icon">edit_icon</string>
 
     <!-- Mission In Progress Activity -->
-    <string name="mission_null">Cannot launch a null mission</string>
+    <string name="mission_invalid">Provided mission arguments are null</string>
```

Comment:
Could maybe be adapted to: "Some of the provided mission arguments are null" Or even better: telling which one is null

Reply:
I added the first text